airflow/migrations/versions/0112_2_4_0_add_dagwarning_model.py

```diff
@@ -44,7 +44,7 @@ def upgrade():
         'dag_warning',
         sa.Column('dag_id', StringID(), primary_key=True),
         sa.Column('warning_type', sa.String(length=50), primary_key=True),
-        sa.Column('message', sa.String(1000), nullable=False),
+        sa.Column('message', sa.Text(), nullable=False),
```

Comment:
@dstandish there's some inconsistency here. In ORM we have `Text` and here is `String`. I think `Text` will be more appropriate? WDYT